SPECS-EXTENDED/libcmpiutil/libcmpiutil.spec

```diff
@@ -0,0 +1,180 @@
+# -*- rpm-spec -*-
+Summary:        CMPI Utility Library
+Name:           libcmpiutil
+Version:        0.5.7
+Release:        20%{?dist}
+License:        LGPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://libvirt.org/CIM/
+Source:         ftp://libvirt.org/libvirt-cim/libcmpiutil-%{version}.tar.gz
```

Comment:
Would it be possible to have a `https` source ?

Reply:
I am now using https instead of ftp. Please look at the latest commit.